Makefile

```diff
@@ -76,12 +76,15 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: $(DOC_FILES) $(FIGURE_FILES)
 	ls -sh $(shell readlink -f $@)
 endif
 
-validate-examples:
+validate-examples: schema/fs.go
 	go test -run TestValidate ./schema
 
-schema-fs:
+schema/fs.go: $(wildcard schema/*.json) schema/gen.go
```

Comment:
I'm not sure how well make will handle this in a fresh Git checkout.  Maybe Travis should touch `schema/gen.go` to ensure a newer-than-`fs.go` mtime?